sites/en/testing-rails-applications/additional_concepts.step

```diff
@@ -19,13 +19,14 @@ message <<-MARKDOWN
   So, when you write a test that calls the title attribute of the orange double, you'll always get back the string Florida Orange. Got it? Good!
 
   ### Spies
-  With spies, we are not talking about espionage... at least, not in relation to testing :) Spies can be used to verify whether a method was called on an object.
+  With spies, we are not talking about espionage... at least, not in relation to testing :) Spies can be used to verify whether a method was called on an object a certain number of times.
   For instance (assume you already have the orange double from above):
 
   ```ruby
   orange = spy('orange')
   orange.name
-  expect(orange).to have_received(:name)
+  orange.name
+  expect(orange).to receive(:name).exactly(2).times
```

Comment:
Do you think it's worth adding the example of using `receive` on it's own and then showing this example of specifying the number of calls?